vulnerabilities/msgvalue-loop.md

```diff
@@ -0,0 +1,44 @@
+# Using ``msg.value`` in a loop.
```

Comment:
This got included in the changes, can you merge/rebase to so that there's no changes made to this file pls

Reply:
Merged msgValue branch. This should fix it.